### ISSUE TITLE
Route Title should be an H1 element

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#15]
The route page title should be an `<h1>`. Check and make sure that no other custom template pages are lacking an `<h1>` as well.
